Added NTP out of sync issue warning (#3303)
aws/eks-anywhere: docs/content/en/docs/tasks/troubleshoot/troubleshooting.md

This is likely a [Memory or disk resource problem]({{< relref "#memory-or-disk-resource-problem" >}}).
You can also try using techniques from [Generic cluster unavailable]({{< relref "#generic-cluster-unavailable" >}}).

### NTP Time sync issues between control plane nodes and worker nodes
```
level=error msg=k8sError error="github.com/cilium/cilium/pkg/k8s/watchers/endpoint_slice.go:91: Failed to watch *v1beta1.EndpointSlice: failed to list *v1beta1.EndpointSlice: Unauthorized" subsys=k8s
```

> You might notice Unauthorized errors if your EKS Anywhere control plane nodes and worker nodes are out of sync. Please ensure to make them configured with same healthy NTP servers to avoid out of sync issues.

Reviewer: ```suggestion
You might notice authorization errors if the timestamps on your EKS Anywhere control plane nodes and worker nodes are out of sync. Please ensure that all the nodes are configured with same healthy NTP servers to avoid out of sync issues.
```

Reviewer 2: Another small suggestion: Change "out of sync issues" to "out-of-sync issues".
